index.html

```diff
@@ -1398,17 +1404,24 @@ <h2>String Matching of Syntactic Content in Document Formats and Protocols</h2>
       <section id="specifying-content-restrictions">
 		  <h2>Specifying Content Restrictions</h2>
 		  
-		  <p>One of the ways in which string matching can be made more effective and consistent is by applying restrictions to the content that is to be matched. Document formats and protocols often do this by defining a namespace or set of rules for what is legal in a given context.</p>
+		  <p>One of the ways in which string matching can be made more effective and consistent is by applying restrictions to the content that is to be matched. When building a <a>vocabulary</a> and especially when defining <a>user-supplied values</a> within that vocabulary, specifications for document formats and protocols often define a namespace or set of rules for what is legal in a given context. This usually includes length and content restrictions.</p>
+		  
+		  <p class="requirement"><span class="qrec">[S]</span> Specifications SHOULD NOT allow the <code>C0</code> (<span class=uname>U+0000</span> to <span class=uname>U+001F</span>) and <code>C1</code> (<span class=uname>U+0080</span> to <span class=uname>U+009F</span>) control characters.</p>
 		  
-		  <p>There are two broad classes that strings or identifier types fall into: those that are meant to be seen, read, or interacted with by humans (and thus might be expected to contain natural language text); and those that are application or protocol internal and not intended for human interaction.</p>
+		  <p>There are two broad classes of identifer: <a>user-facing identifiers</a> and <a>application internal identifiers</a>.</p>
```

Comment:
"identifer" should be "identifier" or "identifiers" here.

Reply:
good catch